plugins/cloudformation/src/index.mjs

```diff
@@ -0,0 +1,62 @@
+const service = 'cloudformation'
+const required = true
+
+/**
+ * CloudFormation plugin maintained by: @architect
+ */
+export default {
+  service,
+  methods: {
+    //
+    // UpdateStack
+    // https://docs.aws.amazon.com/AWSCloudFormation/latest/APIReference/API_UpdateStack.html
+    //
+    // DescribeStacks
+    // https://docs.aws.amazon.com/AWSCloudFormation/latest/APIReference/API_DescribeStacks.html
+    //
+    // DescribeStackEvents
+    // https://docs.aws.amazon.com/AWSCloudFormation/latest/APIReference/API_DescribeStackEvents.html
+    //
+    CreateStack: {
+      awsDoc: 'https://docs.aws.amazon.com/AWSCloudFormation/latest/APIReference/API_CreateStack.html',
+      validate: {
+        StackName: {
+          type: 'string', 
+          required, 
+          comment: "A region unique name associated with the stack."
+        },
+        TemplateBody: { 
+          type: 'string', 
+          required
+        },
+        Capabilities: {
+          type: 'array', 
+          comment:'https://docs.aws.amazon.com/AWSCloudFormation/latest/APIReference/API_CreateStack.html' 
+        },
+      },
+
+      async request ({ StackName, TemplateBody,  Capabilities=['CAPABILITY_IAM', 'CAPABILITY_AUTO_EXPAND']}, {region}) {
+        return {
+          endpoint: `https://${ service }.${ region }.amazonaws.com/`
+          query: {
+            StackName, TemplateBody, Capabilities
+          }
+        }
+      },
+
+      async response (raw) {
+        console.log('got response', raw)
+        let response = raw.Item || null
+        return { response }
```

Comment:
This is the old `response()` semantics; if you just want to see the raw requests + responses passing through and only return the response payload (and not the status code + headers), turn on debug mode (`await awsLite({ debug: true })`) and instead do: `response: async ({payload}) => payload`

Reply:
Template fixed here: https://github.com/architect/aws-lite/pull/20/commits/43f2de5d44014f90e670f9b32d21e7e09bca7dbb